### PR TITLE
OSSM-6174 OSSM 2.5 release notes should indicate support from 4.12 and later

### PR DESCRIPTION
**Pull request by gwynnemonahan**
[OSSM-6174](https://issues.redhat.com//browse/OSSM-6174) OSSM 2.5 release notes should indicate support from 4.12 and later

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSSM-6174

Link to docs preview:
https://73816--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes

QE review:
QE approval is not needed for this PR.

Additional information:

Also update language for the OSSMC plugin to be consistent with the language used elsewhere, and it seems changing a setting in VSCode caught extra spaces that seems to have been automatically removed in this PR.

### CODE REVIEW

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

File: modules/ossm-rn-new-features.adoc [hunk 1]
```diff
@@ -55,28 +55,28 @@ This release adds documentation for migrating from a multitenant mesh to a clust
...
+{DTProductName} (Tempo) Stack is not supported on {ibm-z-title}.
 ====
 
 === OpenShift Service Mesh Console plugin
 
-This release introduces a generally available version of the OpenShift {SMProductShortName} Console (OSSMC) plugin. 
+This release introduces a generally available version of the OpenShift {SMProductShortName} Console (OSSMC) plugin.
 
-The OSSMC plugin is an extension to the OpenShift Console that provides visibility into your Service Mesh. With the OSSMC plugin installed, a new Service Mesh menu option is available in the left-hand navigation of the web console, as well as new Service Mesh tabs that enhance existing Workloads and Service console pages. 
+The OSSMC plugin is an extension to the OpenShift Console that provides visibility into your Service Mesh. With the OSSMC plugin installed, a new Service Mesh menu option is available on the left side of the web console, as well as new Service Mesh tabs that enhance existing Workloads and Service console pages.
```
Comment (stevsmit) [inline modules/ossm-rn-new-features.adoc:73]: We should revise this, based on this role for "navigation" panes: https://redhat-documentation.github.io/supplementary-style-guide/#accessibility-visual-info

maybethis, or something like it:

```suggestion
The OSSMC plugin is an extension to the OpenShift Console that provides visibility into your Service Mesh. With the OSSMC plugin installed, a new Service Mesh menu option is available on the navigation pane of the web console, as well as new Service Mesh tabs that enhance existing Workloads and Service console pages.
```

**Pull request merged**